README.md

```diff
@@ -2270,6 +2460,47 @@ condition](#safe-assignment-in-condition).
   end
   ```
 
+* <a name="duck-typing"></a>
```

Comment:
True, the example that is provided does not really show what advantages duck-typing has over inheritance. Although I do agree that one should make a conscious choice on whether inheritance is the most appropriate solution, since there are modules, duck-typing, composition to use as well.

 I could add a linkt to wikipedia, this is pretty well known guideline (actually I could probalbly even come up with a really easy example): https://en.wikipedia.org/wiki/Composition_over_inheritance